docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -614,15 +619,21 @@ code: |
       progress += increment
       logic_list.append("set_progress(" + str(round(progress,2)) +")")    
     logic_list.append(field.trigger_gather)
+    
+    if field.trigger_gather.endswith('.signature'):
+      signatures.add(field.trigger_gather)
   
-  # Save a snapshot of interview answers. 
-  # We only want a few anonymous variables
-  logic_list.append("# Save (anonymized) interview statistics.")
-  logic_list.append("store_variables_snapshot(data={'zip': users[0].address.zip})")
```

Comment:
I think I missed this convo when it happened: https://github.com/SuffolkLITLab/docassemble-assemblylinewizard/issues/153#issuecomment-774731126. IMO we would want this to _not_ be in the umbrella interview, especially if we want to see how many people are using a specific form. For example, if we made an optional indigency form that was used in several other forms, X and Y, we'd want to know how many people actually used the indigency form, not just how many filled out X and Y, right?

This shouldn't hold up this PR though.

Reply:
Issue #223 made!